client/lib/products-values/constants.js

```diff
@@ -200,34 +202,17 @@ export const getJetpackProducts = () => {
 			slugs: JETPACK_BACKUP_PRODUCTS,
 		},
 	];
-	isEnabled( 'jetpack/scan-product' ) &&
-		output.push( {
-			title: translate( 'Jetpack Scan' ),
-			// TODO: Add new description copy for Search
-			description: 'Always-on scan ensure you never lose your site.',
-			id: PRODUCT_JETPACK_SCAN,
-			forceRadios: true,
-			options: {
-				yearly: [ PRODUCT_JETPACK_SCAN ],
-				monthly: [ PRODUCT_JETPACK_SCAN_MONTHLY ],
-			},
-			optionShortNames: getJetpackProductsShortNames(),
-			optionDisplayNames: getJetpackProductsDisplayNames(),
-			optionDescriptions: getJetpackProductsDescriptions(),
-			optionsLabel: translate( 'Select a product option:' ),
-			slugs: JETPACK_SCAN_PRODUCTS,
-		} );
 	isEnabled( 'jetpack/search-product' ) &&
 		output.push( {
 			title: translate( 'Jetpack Search' ),
-			// TODO: Add new description copy for Search
 			description: translate(
-				'Always-on backups ensure you never lose your site. Choose from real-time or daily backups.'
+				'Incredibly powerful and customizable, Jetpack Search helps your visitors instantly find the right content – right when they need it.'
 			),
```

Comment:
How about this instead?

```jsx
			description: getJetpackProductsDescriptions()[ PRODUCT_JETPACK_SEARCH ],
```

Reply:
Yes, of course! That makes a lot of sense.